### DISCUSSION TITLE
How to add costum labels?

### DISCUSSION THREAD

**Discussion by LeifHenriksen** [kurkle/chartjs-chart-treemap#56] [Q&A] [answered]
Hi,

How do you add custom labels (I don't know if they are called labels)?
As of today the label value is the same as the **groups** value, but I would modify modify it while conserving the same groups.

![image](https://user-images.githubusercontent.com/44870274/141697654-8200f13d-26cf-4e6c-b89b-a71553f535f7.png)
 
For example, in the image above I would like to add some text after **FL** or **TX**, how can I do that?

Thanks in advance.

Example code pen  : https://codepen.io/kurkle/pen/JqbzgQ

**Comment by stockiNail** [comment]
The capability is ready in the `master`. See PR #48.

Have a look to the doc as well: https://chartjs-chart-treemap.pages.dev/usage.html#labels

**Reply by LeifHenriksen** [reply]
Ok, so I tried using the **formatter** and it works in the docs  https://chartjs-chart-treemap.pages.dev/usage.html#labels, if I add 
```js
labels: {
         display: true,
         formatter: (ctx) => 'Kmq ' + ctx.raw.v
}
```
I get the correct result.

![image](https://user-images.githubusercontent.com/44870274/141702622-8f8b7e45-66fe-41b0-bdcd-143f9a61475d.png)

But when I try to use it in my project or inside a codepend it doesn't work, this codepen has the exact same code that works in the docs site but not inside the codepen (There are no labels) https://codepen.io/leifhenriksen/pen/porxqar

I am using chartjs 3.6.0 and chartjs-treemap 1.0.2 inside the codepen, so I really don't know where the issue is coming from.

**Reply by kurkle** [reply]
This feature will be released in next version, so it's not yet available in npm.

**Comment by stockiNail** [comment, marked as answer]
@LeifHenriksen the PR has been merged in the master but not released yet. Therefore, using version 1.0.2, unfortunately you don't have that feature. 
AFAIK, it's planned to be released by version 2.0, [see Milestone 2.0](https://github.com/kurkle/chartjs-chart-treemap/milestone/2?closed=1)

**Reply by LeifHenriksen** [reply]
Ok, thank you. 

I used npm to create a dist file from master and it is working.
